logs/archived/spring-boot-logger-2022-07-25.0.log

```diff
@@ -0,0 +1,78 @@
+2022-07-25 11:46:02,272 INFO org.springframework.boot.StartupInfoLogger [restartedMain] Starting PaketomatApplication using Java 17.0.3.1 on DESKTOP-2HFIQUK with PID 11860 (C:\Users\StefanTomic\Documents\paketomat\target\classes started by StefanTomic in C:\Users\StefanTomic\Documents\paketomat)
```

Comment:
Please delete this file.

Reply:
Done.

Reply:
I can still see this file, maybe you forgot to commit it.

Reply:
Done.